### PR TITLE
NOISSUE - Correct readers openapi.yml

### PR DESCRIPTION
**Pull request by manuio**
Signed-off-by: Manuel Imperiale <manuel.imperiale@gmail.com>

### CODE REVIEW

**Review by drasko — APPROVED** [review]
LGTM

File: readers/openapi.yml [hunk 1]
```diff
@@ -123,6 +125,21 @@ components:
...
+      schema:
+        type: string
+        format: uuid
+      required: false
+    Name:
+      name: name
+      description: SenML message name.
+      in: query
+      schema:
+        type: string
```
Comment (darkodraskovic) [inline readers/openapi.yml:141]: "OpenAPI 3 introduces support for objects as query parameters." Please see here https://www.baeldung.com/openapi-json-query-parameters, specifically here https://www.baeldung.com/openapi-json-query-parameters#OAP3

**Review by darkodraskovic — CHANGES_REQUESTED** [review]

File: readers/openapi.yml (hunk 1 above)
Comment (manuio) [reply readers/openapi.yml:141]: @darkodraskovic Looks like this explains more how to generate an API that accept JSON objects. AFAIK we don't support this in our services. As you previously did I would follow: https://swagger.io/docs/specification/describing-parameters/

Comment (darkodraskovic) [reply readers/openapi.yml:141]: This also allows for automatic Postman collections etc. I'm not sure, but it seams like a standard way to specify json string in query. Just add
```
        schema:
          type: object
```
And see if the editor complains.

**Review by darkodraskovic — APPROVED** [review]

**Pull request merged**